docker-compose.yml

```diff
@@ -697,7 +715,7 @@ services:
       networks:
         - frontend
         - backend
-        
```

Comment:
can you please revert all of those whitespace removing lines back?
This PR is small, but it's definitely  easier to look at PR without such things